api4/team.go

```diff
@@ -544,6 +598,11 @@ func addTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	auditRec := c.MakeAuditRecord("addTeamMembers", audit.Fail)
+	defer c.LogAuditRec(auditRec)
+	auditRec.AddMeta("team_id", c.Params.TeamId)
+	auditRec.AddMeta("count", len(members))
```

Comment:
Should we add the user ids to the audit record here?

Reply:
Agreed.  Done via https://github.com/mattermost/mattermost-server/pull/13967/commits/707fc3bcdfbc4d8bc4a3db1f710f96fd83311763